src/vs/workbench/contrib/welcome/page/browser/vs_code_welcome_page.ts

```diff
@@ -21,7 +21,8 @@ export default () => `
 						<li><a href="command:workbench.action.files.newUntitledFile">${escape(localize('welcomePage.newFile', "New file"))}</a></li>
 						<li class="mac-only"><a href="command:workbench.action.files.openFileFolder">${escape(localize('welcomePage.openFolder', "Open folder..."))}</a></li>
 						<li class="windows-only linux-only"><a href="command:workbench.action.files.openFolder">${escape(localize('welcomePage.openFolder', "Open folder..."))}</a></li>
-						<li><a href="command:workbench.action.addRootFolder">${escape(localize('welcomePage.addWorkspaceFolder', "Add workspace folder..."))}</a></li>
+						<li><a href="command:workbench.action.openWorkspace">${escape(localize('welcomePage.openWorkspace', "Open workspace..."))}</a></li>
```

Comment:
I'm not sure we'd want to add this option as it would be the length longer. Plus, when you choose "Open Folder" you can already choose to open a workspace.

Reply:
When I use "Open folder..." on Windows I get the native "Open Folder" dialog, which means there's no way I can choose a .code-workspace file.

Can we allay your concern about making the length longer by reducing my a corresponding amount the large amount of empty space above the "Recent" header?